include/can_interface.h

```diff
@@ -345,6 +436,70 @@ class ICAN
     virtual void Tick() = 0;
 };
 
+class MockCAN : public ICAN
+{
+public:
+    void Initialize(BaudRate baud __attribute__((unused))) {}
+    bool SendMessage(CANMessage &msg)
```

Comment:
Is that section supposed to be commented out? What's the point of this class implementation?

Reply:
This is for use in testing by mocking the CAN without having to access hardware